gcp/modules/gcp-stackdriver-monitoring/resources/alert_policies/couchdb_prometheus_exporter_exports_metric.json

```diff
@@ -27,7 +27,7 @@
           }
         ]
       },
-      "display_name": "Check for absense of custom/couchdb/httpd_node_up metric"
+      "display_name": "custom/couchdb/httpd_node_up"
```

Comment:
I liked your improvement to this one. :)

Reply:
Yeah, I added this to GPII-3942.